File: 01-backend/spring-boot-library/src/main/java/com/libranova/spring_boot_library/dto/response/UserLoansSummary.java
package com.libranova.spring_boot_library.dto.response;

import com.libranova.spring_boot_library.model.Book;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class UserLoansSummary
{
    private Book book;

    private int daysRemaining;

}

File: 02-frontend/react-library-app/src/models/UserLoansSummary.ts
import exp from "constants";
import Book from "./Book";

class UserLoansSummary{
    book: Book;
    daysRemaining: number;

    constructor(book: Book, daysRemaining: number) {
        this.book = book;
        this.daysRemaining = daysRemaining;
    }
}

export default UserLoansSummary;